Makefile

```diff
@@ -7,12 +7,19 @@ $(error $(CONFIG_FILE) not found. See $(CONFIG_FILE).example.)
 endif
 include $(CONFIG_FILE)
 
+# Rectify input parameters
+ifeq ($(CPU_ONLY),1)
+  USE_CUDNN=0
+endif
+
+PROJECT_DIR=$(PWD)
```

Comment:
What's the reason for doing this? Is this necessary for the cuDNN changes, or can we do this in another PR (if it's really needed)?